tasks/Task.cpp

```diff
@@ -92,10 +90,20 @@ bool Task::configureHook()
     }
     setDriver(driver.get());
 
+    if (!TaskBase::configureHook()) {
+        return false;
+    }
+
     mDestinationId = _destination_id.get();
     mMsgType = _msg_type.get();
-
-    if (!TaskBase::configureHook()) {
+    m_orientation_output_flag = _orientation_output_flag.get();
+    m_ping_refresh_period = _ping_refresh_period.get();
+
+    if (m_orientation_output_flag &&
+        MINIMUM_PING_STATUS_REFRESH_TIME > m_ping_refresh_period) {
+        LOG_ERROR_S << "If an orientation is required, because of hardware limitations, "
+                       "the minimum ping refresh time is : "
+                    << MINIMUM_PING_STATUS_REFRESH_TIME << std::endl;
```

Comment:
Change the message:
Due to hardware limitations, the minimum ping update rate required to get orientation is: